recap/readers/postgresql.py

```diff
@@ -33,6 +33,10 @@ def get_recap_type(self, column_props: dict[str, Any]) -> RecapType:
             base_type = StringType(bytes_=octet_length, variable=True)
         elif data_type.startswith("char"):
             base_type = StringType(bytes_=octet_length, variable=False)
+        elif data_type == "uuid":
+            base_type = StringType(
+                logical="build.recap.UUID", bytes_=36, variable=False
```

Comment:
Nit: mind adding a trailing `,` and running `pdm run style` again?

Reply:
Done!